### ISSUE TITLE
Passing weaver flags 

### ISSUE THREAD

**Issue by errordeveloper** [weaveworks/docker-plugin#8]
Currently plugin only passes peers and any other flags are treated as unrecognised. One good example of flag that someone is quite likely to use is `-initpeercount N`.

**Comment by squaremo** [comment]
Here it would be nice to have proper unix-y behaviour from the flag parser; i.e., to be able to partition arguments for the weave container with a `--`.

**Comment by squaremo** [comment]
After #13, you can pass these as `$WEAVE_ARGS`, or ignore the script and start weave and the plugin with your own command lines.
